core/server/apps/default-cards/cards/markdown.js

```diff
@@ -1,16 +1,44 @@
 var SimpleDom   = require('simple-dom'),
     tokenizer   = require('simple-html-tokenizer').tokenize,
+    jsdom       = require('jsdom').jsdom,
```

Comment:
> Could you please move all third party dependencies into the render function?

👍 

> Furthermore, the global variables e.g. html, doc can be local variables.

I tried that but JSCS was complaining so thought it was some node rule I wasn't aware of 😄. I'll make them local and add an ignore if I can't get JSCS to behave